src/LemonBot.Commands/CommandDescriptor.cs

```diff
@@ -2,22 +2,22 @@
 
 public record CommandDescriptor
```

Comment:
We could transform this record with the property construct. In this case all the properties won't give null warning anymore